patroni/postgresql/misc.py

```diff
@@ -2,13 +2,38 @@
 import logging
 import os
 
+from enum import Enum
 from typing import Iterable, Tuple
 
 from ..exceptions import PostgresException
 
 logger = logging.getLogger(__name__)
 
 
+class PostgresqlState(str, Enum):
+    """Possible values of :attr:`Postgresql._state`."""
```

Comment:
Lest better refer to a public attribute.
```suggestion
    """Possible values of :attr:`Postgresql.state`."""
```